Model/ResourceModel/Events/Collection.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Klaviyo\Reclaim\Model\ResourceModel\Events;
+
+use Magento\Framework\Model\ResourceModel\Db\Collection\AbstractCollection;
+
+class Collection extends AbstractCollection
+{
+    protected $_idFieldName = 'id';
+
+    /**
+     * Define resource model
+     *
+     * @return void
+     */
+    protected function _construct()
+    {
+        $this->_init( 'Klaviyo\Reclaim\Model\Events', 'Klaviyo\Reclaim\Model\ResourceModel\Events' );
+    }
+
+    public function getEventsToUpdate()
+    {
+        $eventsCollection = $this->addFieldToSelect( ['id','event','payload','user_properties'] )
+            ->addFieldToFilter( 'status','New' )
```

Comment:
```suggestion
            ->addFieldToFilter( 'status','NEW' )
```
We really need to have a consistent naming convention.  In most places I see it in ALL Caps